cmd/kubeadm/app/master/apiclient.go

```diff
@@ -68,9 +68,12 @@ func CreateClientAndWaitForAPI(file string) (*clientset.Clientset, error) {
 	fmt.Println("[apiclient] Created API client, waiting for the control plane to become ready")
 
 	start := time.Now()
-	wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
+	err = wait.PollInfinite(apiCallRetryInterval, func() (bool, error) {
 		cs, err := client.ComponentStatuses().List(v1.ListOptions{})
```

Comment:
+1 on switching to /healthz endpoints. I thought the component status api was a black sheep. cc @bgrant0607